test/functional/page_objects/home_page.ts

```diff
@@ -65,6 +63,7 @@ export class HomePageObject extends FtrService {
     // where it appears the click just didn't work.
     await this.common.sleep(1010);
     await this.testSubjects.click(`removeSampleDataSet${id}`);
+    await this.common.sleep(1010);
     await this._waitForSampleDataLoadingAction(id);
```

Comment:
We're waiting for the loader to no longer be present, so in some rare case, the loader is not yet present when we enter the retry loop, causing the function to resolves too early

Reply:
> in some rare case

Do we have any idea what the rare case is? If not, should we be investigating this?